Interfaces/API/Job.py

```diff
@@ -554,26 +556,83 @@ def setDestination(self, destination):
     return S_OK()
 
   #############################################################################
-  def setNumberOfProcessors(self, processors):
+  def setNumberOfProcessors(self, processors=None, minNumberOfProcessors=None, maxNumberOfProcessors=None):
```

Comment:
I would also call the first argument numberOfProcessors rather than just processors